src/server/controllers/ApiController.js

```diff
@@ -1,42 +1,70 @@
-import User from '../models/user';
-import { response } from '../helpers/errorHandler';
+import User from '../models/user'
+import { response } from '../helpers/errorHandler'
+import { generate } from '../helpers/token'
+
+const availableTokens = []
 
 export default class ApiController {
-  static index (req, res) {
-    res.json({ version: '0.0.1' });
-  }
   static postRegistrationHandler (req, res, next) {
-    const { username, email, password } = req.body;
-    const user = new User({ username, email, password });
+    const { username, email, password } = req.body
+    const user = new User({ username, email, password })
     User
       .findOne({ email })
       .then(foundUser => foundUser
         ? response(res, 'User already exists!', 404)
         : saveUserInDB(res, user))
-      .catch(err => response(res, err.message, 404));
+      .catch(err => response(res, err.message, 404))
   };
+
   static postLoginHandler (req, res, next) {
-    const { email, password: receivedPassword } = req.body;
+    const { email, password: receivedPassword } = req.body
     User
       .findOne({ email })
       .then(foundUser => foundUser
         ? passwordComparison(res, foundUser, receivedPassword)
         : response(res, 'Login failed', 404))
-      .catch(err => response(res, err.message, 401));
+      .catch(err => response(res, err.message, 401))
   };
+
+  /**
+   *
+   * @param {string} refresh
+   * @param {string} userId
+   */
+  static refreshTokenHandler (req, res) {
+    const { refresh, userId } = req.body
+
+    const findedRefreshToken = availableTokens
+      .find((token, index) => token.ref === refresh)
```

Comment:
You can remove index param from this place if you don't used

Reply:
added

Reply:
fixed